### DISCUSSION TITLE
ethereum-lists chains and CASA

### DISCUSSION THREAD

**Discussion by bmann** [ChainAgnostic/CASA#58] [General]
This is sort of meta, but also relevant as it crosses over some of the same people / mission

@pedrouid said that ethereum-lists should be something that the EF takes over 

I'm one of the people with permissions on https://github.com/ethereum-lists/chains. Right now, @ligi pretty much maintains this himself on an as time permits. I've had the situation where people chase me down in DMs in various places to get 

So: @ligi @pedrouid should ethereum-lists be something maintained by CASA?

If not, what is the next action to eg. transfer to EF? Or @ligi do you want to keep doing it all yourself.

I've removed myself as an admin from the https://github.com/ethereum-lists/chains in the meantime

**Comment by ligi** [comment]
I am happy about help here - but:
 * not sure if "the EF takes over" is in the spirit of subtraction of the EF and so really question if "the EF" would or even should do it
 * it uses CASA standards but not sure it should be maintained by CASA - IMHO we should also live the spirit of subtraction in CASA and not pile up projects the alliance maintains but concentrate on the core mission: standards

**Reply by bmann** [reply]
OK, so unless someone else feels strongly about this, this is a Ligi and whoever helps him side project. Somewhere in the relevant CAIPs (or maybe nowhere?) -- we say go register EVM compatible chainIDs --> `ethereum-lists/chains`

Re: substraction for CASA. If the net outcome is organizational overhead of various people running side projects that are critically relied on by CASA, doesn't seem like a good outcome. Lists of normative and widely used chain IDs feels like an important standard.

That's all I have to say, and others can revisit this thread as needed if it wants to be discussed further.

**Reply by ligi** [reply]
I do not think ever growing lists are a standard - lists can use standards for structure and interoperability but they are no standard them self.

**Reply by bmann** [reply]
Sure. At some point, when lists become widely used, they become load bearing. And maybe this isn't and it's fine. It's probably the distinction between "just a list" and "a registry". e.g. IANA maintains a registry and has a process for adding to it https://www.iana.org/protocols

**Comment by ritave** [comment]
I'd also like the ethereum-list to be outside of CASA, so that we can focus more on building actually used standards, since it's already a hard thing to succeed with without any distractions.
